src/panel/direction/RouteVia.jsx

```diff
@@ -7,22 +7,22 @@ const RouteVia = ({ route, vehicle }) => {
     return <div className="routeVia">
       {_('Via', 'direction')} {route.legs[0].summary.replace(/^(.*), (.*)$/, '$1')}
     </div>;
+  } else {
```

Comment:
Why this else? The early return construct was on purpose.

Reply:
the else has always been there in the file's history, but ok :)

Reply:
Ah? In that case I'm sorry. I just didn't understand these lines of diff which did nothing, but I may have misread it.

Reply:
np, I had just added a superfluous test ("else if(...) { "), instead of the "else" that was already here, and finally removed it